pallets/staking-precompiles/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "parachain-staking-precompiles"
```

Comment:
As you know, `Pallet`  is a substrate module. I'd prefer to have one folder just for substrate modules (`pallets/`) and one for the precompiles. Could we just add a `precompiles` folder to the root instead of putting it in `pallets/`?

Reply:
Yeah, I knew that wasn't quite the right place for it... I'll fix it :)

Reply:
b5d5b0